ui/src/preview/change-table-sorting.ts

```diff
@@ -9,13 +13,6 @@ export enum SortProps {
     CHANGE_TIME = 'CHANGE_TIME',
     USER_NAME = 'USER_NAME',
     ERRORS = 'ERRORS',
-    PUSHED_TO_RATKO = 'PUSHED_TO_RATKO',
-}
-
-export enum SortDirection {
```

Comment:
Siirretty publicationin alle

Reply:
Kokonaisuudessaan hyviä refakkeja 👏 En ollut kauhean tyytyväinen siihen mihin tilaan just nämä yhteiset change-table-tiedostojen alle jääneet jutut jäi silloin kun ne tein. Kysymyksenä tosin, täällä oli paljon sekä esikatselunäkymän että etusivun julkaisudetaljinäkymän käyttämää tarvaraa ja ne oli vähän niinkuin tarkoituksella muualla kuin publicationin alla, koska itse mielsin publication-termin käytännössä tässä yhteydessä etusivun julkaisudetaljinäkymäksi. Toimiihan molempien taulukoiden sorttaus edelleen?

Reply:
Detaljit on duplikoitu molempiin filuihi (not nice).
En oo ihan tyytyväinen siihen, että nyt on kahdessa paikkaa lähes samat koodit, mutta halusin erottaa näkymät toisistaan.

Ois varmaan tarvetta geneeriselle sorttauskomponentille.